scripts/cryptoGenDocker.sh

```diff
@@ -0,0 +1,55 @@
+#! /bin/bash
```

Comment:
The usage pattern is confusing for me. Why do we have two `cryptogen` scripts. For users, having one would avoid confusion.

Reply:
Ok. It started with one script - `cryptoGen.sh`. But, because some differences in Mac, it was failing and required extra configuration of openssl on Mac. So, I created script that use dockerized openssl, but kept original script as well. If you think we don't need both, I can remove original script.

Reply:
In that case, rather than using two scripts, we can use a single script and pass the machine architecture as an input. Otherwise, we can just use `docker` script alone removing the old script. I leave the choice to you.